check/create-check-api.js

```diff
@@ -0,0 +1,33 @@
+const check = require('./check');
+const validator = require('validator');
+const {
+  buildCheckFunction
+} = require('./validation-chain-builders');
+
+const createCheckAPI = ({
+  customValidators,
+  customSanitizers
+} = {}) => {
+  const customCheck = (fields, locations, message) =>
+    check(
+      fields,
+      locations,
+      message,
+      Object.assign({}, customValidators, validator),
+      Object.assign({}, customSanitizers, validator)
```

Comment:
By doing this, one could have their custom validators/sanitizers overwritten by validator's ones.
Could be an astonishment point for users that deliberately wanted to overwrite the standard ones, or for those that didn't know one with the same name exists and so their function never runs.